### DISCUSSION TITLE
Workshop: REST with Jakarta EE

### DISCUSSION THREAD

**Discussion by ecabrerar** [JavaDominicano/java-dominicano-comite-ejecutivo#45] [Ideas]
# REST with Jakarta EE

## Abstract
In this workshop,you will get an introduction to the new and updated APIs of the Jakarta EE 10 Platform. You will get hands-on experience of the Platform with practical exercises of selected Jakarta EE 10 APIs.

The focus of the workshop will be to build RESTful Web Services with Jakarta EE. You will learn how to deploy your services to a traditional application server as well as how to create a bootable JAR with the SEBootstrap mechanism provided by Jakarta EE RESTful Web Services.

You will learn how to leverage the power of the Jakarta EE APIs for serializing JSON to Java objects and back. All in all everything you need to know to be successful creating RESTful Web Services with Jakarta EE.

**Comment by ecabrerar** [comment]
@fredpena @BrayanMnz I have a couple suggestions.

1. Change the title: Could be **Unlock the potential of Jakarta REST to build RESTful Web Services**
2. Add the following text before the last paragraph:
**Did you know you don't need an external library like Jackson or Gson for JSON processing?**

**Comment by ecabrerar** [comment]
**Putting all together:**

# Unlock the potential of Jakarta REST to build RESTful Web Services

## Abstract
In this workshop,you will get an introduction to the new and updated APIs of the Jakarta EE 10 Platform. You will get hands-on experience of the Platform with practical exercises of selected Jakarta EE 10 APIs.

The focus of the workshop will be to build RESTful Web Services with Jakarta EE. You will learn how to deploy your services to a traditional application server as well as how to create a bootable JAR with the SEBootstrap mechanism provided by Jakarta EE RESTful Web Services.

Did you know you don't need an external library like Jackson or Gson for JSON processing?

You will learn how to leverage the power of the Jakarta EE APIs for serializing JSON to Java objects and back. All in all everything you need to know to be successful creating RESTful Web Services with Jakarta EE.

**Comment by fredpena** [comment]
Everything sound great!
